content/en/coec/index.adoc

```diff
@@ -0,0 +1,53 @@
+---
+title: "Code of Ethics and Conduct"
+description: "ACL Special Interest Group on Turkic Languages Code of Ethics and Conduct"
+draft: false
+toc: true
+images: []
+contributors: []
+---
+
+:toc:
+
+== Introduction
+
+SIGTURK is an ACL Special Interest Group on Turkic Languages that aims to foster community brainstorming and collaboration on events and resources for the public good in open source. As these processes involve sync and async communication, this document includes professional standards all participants of SIGTURK and SIGTURK-related dialogue must uphold to promote formal reliability of SIGTURK outputs, respect, competence, responsibility, and integrity while recognizing the rapidly changing society and the necessity to exercise more considerable caution in this spirit by every participant.
+
+SIGTURK also recognizes the following documents as authoritative, and their reading should interpret the authoring organization's name as SIGTURK under all respect to the license of the individual page:
+
+* ACL Code of Ethics: https://www.aclweb.org/portal/content/acl-code-ethics
+* ACL Anti-Harassment Policy: https://www.aclweb.org/adminwiki/index.php?title=Anti-Harassment_Policy
+* ACM Code of Ethics and Professional Conduct: https://www.acm.org/code-of-ethics
+* W3C Code of Ethics and Professional Conduct: https://www.w3.org/Consortium/cepc
+* GitHub Community Code of Conduct: https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct
+* RISC-V Community Code of Conduct: https://riscv.org/community/community-code-of-conduct/
+* SIL Basic Principles of Ethical Research in Fieldwork: https://www.sil.org/about/discover/academic-research
+
+These organizations and codes of ethics and conduct are model documents for SIGTURK with authoritative relevance.
```

Comment:
Does this mean that our CoEC is modeled after the documents above? If so I think we could word this a bit clearer.

Reply:
Preceding paragraph declares recognition and following paragraph acknowledges the sources of inspiration. I tried to make wording a bit more direct.